spec/Gaufrette/Adapter/FlysystemSpec.php

```diff
@@ -31,11 +31,46 @@ function it_reads_file(AdapterInterface $adapter)
         $this->read('filename')->shouldReturn('Hello.');
     }
 
+    function it_throws_file_not_found_exception_when_trying_to_read_an_unexisting_file(AdapterInterface $adapter)
+    {
+        $adapter->read('filename')->willThrow(new FileNotFoundException('filename'));
+
+        $this->shouldThrow('Gaufrette\Exception\FileNotFound')->duringread('filename');
```

Comment:
> I ran into issues when trying to run the phpspec tests : the phpspec.yml file has a dependency on Akeneo\SkipExampleExtension but I couldn't get it installed via composer (unresolvable set of packages). So I had to comment that file to run the tests on my machine :confused:

Mh, that's strange! The `akeneo/phpspec-skip-example-extension` package [is in our composer.json](akeneo/phpspec-skip-example-extension) for some time now. Was there any error message/stacktrace?

> It would be a good point to bump up the tests tools to their latest version :wink:

Yep, I already upgraded phpunit to 5.7 and there's a task about upgrading phpspec in the github project ;)